include/mbedtls/bignum.h

```diff
@@ -105,9 +106,15 @@
  *  MBEDTLS_MPI_RW_BUFFER_SIZE = ceil(MBEDTLS_MPI_MAX_BITS / ln(10) * ln(2)) +
  *                                LabelSize + 6
  */
-#define MBEDTLS_MPI_MAX_BITS_SCALE100          ( 100 * MBEDTLS_MPI_MAX_BITS )
+#define MBEDTLS_MPI_MAX_BITS_SCALE100          (100 * MBEDTLS_MPI_MAX_BITS)
 #define MBEDTLS_LN_2_DIV_LN_10_SCALE100                 332
-#define MBEDTLS_MPI_RW_BUFFER_SIZE             ( ((MBEDTLS_MPI_MAX_BITS_SCALE100 + MBEDTLS_LN_2_DIV_LN_10_SCALE100 - 1) / MBEDTLS_LN_2_DIV_LN_10_SCALE100) + 10 + 6 )
+#define MBEDTLS_MPI_RW_BUFFER_SIZE             (((MBEDTLS_MPI_MAX_BITS_SCALE100 \
```

Comment:
Is there anything we can do about such macros? This approach seems a bit hardcore.

Reply:
Isn't this just a consequence of line length limits?

Reply:
The smart thing would be to break the line after the macro name. Uncrustify apparently doesn't do this, but it would preserve the line break if there was one (see for example `include/psa/crypto_sizes.h`). I think in general, if we want nicer-looking code, we need to get into the habit of breaking long macro definitions after the macro name.